docs/modules/ROOT/pages/jet-engine-configuration.adoc

```diff
@@ -0,0 +1,72 @@
+= Configuring the Jet Engine
+:description: This section describes Hazelcast’s Jet engine specific configuration using Hazelcast Platform Operator.
+
+{description}
+
+== Configuring the Jet Engine
+When you run Hazelcast using Platform Operator, Jet Engine is enabled by default.
```

Comment:
```suggestion
The Jet engine is enabled by default and has the following configuration options.
```
As we're in the Operator docs, I don't think we need to repeat Hazelcast Platform Operator.